examples/19-EnsembleModel-examples.ipynb

```diff
@@ -0,0 +1,868 @@
+{
```

Comment:
<p>If not specified, a linear regression is used. Using a sklearn model is easy out-of-the-box, but using a <code>RegressionModel</code></p><p>...  -&gt; If not specified, Darts' <code>LinearRegressionModel</code> is used. Using a sklearn model is easy out-of-the-box, but using one of Darts' regression models ...</p><p> </p><p> </p><p> </p><p>Other comment: it sounds like it accepts only these three parameters. Maybe we can write:</p><p> </p><p>It accepts three parameters: -&gt; The main three parameters are:</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1815/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>